app/components/dropdown-search.js

```diff
@@ -0,0 +1,134 @@
+import Ember from "ember";
+import { singularize, pluralize } from "ember-inflector";
+import _ from "lodash";
+const { getOwner } = Ember;
+
+export default Ember.Component.extend({
+  selected: [],
+  previousValue: "",
+  store: Ember.inject.service(),
+  subformDetailService: Ember.inject.service(),
+  resourceType: Ember.computed.alias("packageDetails"),
+  fixedDropdownArr: ["frequency", "voltage", "compTestStatus", "testStatus"],
+  selectedData: Ember.computed.alias("selectedValue"),
+  selectedDataDisplay: Ember.computed.alias("selectedValuesDisplay"),
+  fixedDropdownArr: ["frequency", "voltage", "compTestStatus", "testStatus"],
```

Comment:
This is repeated. Line 12: https://github.com/crossroads/stock.goodcity/pull/535/files#diff-0046bcc5c5d0932112242a81a9ea80c1R12